rllib/env/meta_env.py

```diff
@@ -0,0 +1,35 @@
+from gym.core import Env
+
+
+class MetaEnv(Env):
+    """
+    Wrapper around OpenAI gym environments and meta-learning environments.
+    
+    Your env must implement this interface in order to be used with MAML.
+    """
+
+    def sample_tasks(self, n_tasks):
```

Comment:
```suggestion
    def sample_tasks(self, n_tasks: int) -> List[int]:
```

Reply:
Same thing above (it really depends on the env, it could also be a dict!)

Reply:
Ok, then you can do this:
```-> List[TaskType]```

and on top of the file define
`TaskType = Any  # Can be different types depending on the env, e.g., int or dict`